apps/dashboard/src/components/workflow-row.tsx

```diff
@@ -219,7 +219,7 @@ export const WorkflowRow = ({ workflow }: WorkflowRowProps) => {
       </WorkflowLinkTableCell>
 
       <WorkflowLinkTableCell workflow={workflow} className="text-foreground-600 min-w-[180px] text-sm font-medium">
-        <TimeDisplayHoverCard date={new Date(workflow.updatedAt)}>
+        <TimeDisplayHoverCard date={new Date(workflow.updatedAt)} className="relative z-10">
```

Comment:
I wonder if it's better to add inside of the time display component so it will fix it on all screens (subscribers, etc.)

Reply:
valid point.
right now i saw that it was broken only on workflows (the activity page was fine) so i decided to do it more specifically for now,

Reply:
FYI the problem was because we made the workflow.TableRow `isolate` to support row Click.

Reply:
done